extensions/wikia/Flags/views/FlagView.class.php

```diff
@@ -26,4 +35,15 @@ public function createWikitextCall( $flagView, $params ) {
 
 		return $viewCall;
 	}
+
+	/**
+	 * Wraps calls of notice templates in a div of a class portable-notices.
+	 * @param array $templateCalls
+	 * @return string
+	 */
+	public function wrapTemplateCalls( Array $templateCalls ) {
+		return \Xml::element( 'div', [
```

Comment:
This is being put into wikitext, right? If so, I'd suggest switching this to `Html::rawElement` since `Xml::element` escapes the contents of the div, otherwise if the template parameter values have HTML that is valid wikitext they will be escaped here before the parser handles them so they'll display escaped.